### DISCUSSION TITLE
hello, I hava a question about the skywalking in cluster mode.

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10618] [Q&A] [answered]
I don't know the structure in cluster mode, is it master&slave？ or just multi-node? 
I don't get this info about this question, thx!

**Comment by Superskyyy** [comment, marked as answer]
@qigeqigeqi See this https://skywalking.apache.org/docs/main/next/en/setup/backend/advanced-deployment
